src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/vilkårsvurdering/preutfylling/PreutfyllBosattIRiketServiceTest.kt

```diff
@@ -0,0 +1,91 @@
+package no.nav.familie.ba.sak.kjerne.vilkårsvurdering.preutfylling
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.familie.ba.sak.cucumber.lagVilkårsvurdering
+import no.nav.familie.ba.sak.datagenerator.lagBehandling
+import no.nav.familie.ba.sak.datagenerator.lagMatrikkeladresse
+import no.nav.familie.ba.sak.datagenerator.lagPersonResultat
+import no.nav.familie.ba.sak.datagenerator.lagTestPersonopplysningGrunnlag
+import no.nav.familie.ba.sak.datagenerator.lagVegadresse
+import no.nav.familie.ba.sak.integrasjoner.pdl.PdlRestClient
+import no.nav.familie.ba.sak.kjerne.autovedtak.fødselshendelse.Resultat
+import no.nav.familie.ba.sak.kjerne.vilkårsvurdering.domene.Vilkår
+import no.nav.familie.kontrakter.felles.personopplysning.Bostedsadresse
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+
+class PreutfyllBosattIRiketServiceTest {
+    private var pdlRestClient: PdlRestClient = mockk(relaxed = true)
+    private var preutfyllBosattIRiketService = PreutfyllBosattIRiketService(pdlRestClient)
+
+    @Test
+    fun `skal lage preutfylt vilkårresultat basert på data fra pdl`() {
+        // Arrange
+        val behandling = lagBehandling()
+        val persongrunnlag = lagTestPersonopplysningGrunnlag(behandling.id)
+        val vilkårsvurdering = lagVilkårsvurdering(persongrunnlag, behandling)
+        val personResultat = lagPersonResultat(vilkårsvurdering = vilkårsvurdering)
+
+        every { pdlRestClient.hentBostedsadresserForPerson(any()) } returns
+            listOf(
+                Bostedsadresse(
+                    gyldigFraOgMed = LocalDate.now().minusYears(4),
+                    gyldigTilOgMed = LocalDate.now().minusYears(3),
+                    vegadresse = lagVegadresse(12345L),
+                ),
+                Bostedsadresse(
+                    gyldigFraOgMed = LocalDate.now().minusYears(3).plusDays(1),
+                    gyldigTilOgMed = LocalDate.now().minusYears(2),
+                    matrikkeladresse = lagMatrikkeladresse(54321L),
+                ),
+                Bostedsadresse(
+                    gyldigFraOgMed = LocalDate.now().minusYears(1),
+                    gyldigTilOgMed = null,
+                    vegadresse = lagVegadresse(98765L),
+                ),
```

Comment:
Det bør testes hvordan funksjonen håndterer flere perioder som har `gyldigTilOgMed = null`
```suggestion
                Bostedsadresse(
                    gyldigFraOgMed = LocalDate.now().minusYears(1),
                    gyldigTilOgMed = null,
                    vegadresse = lagVegadresse(98765L),
                ),
                Bostedsadresse(
                    gyldigFraOgMed = LocalDate.now().minusMonths(6),
                    gyldigTilOgMed = null,
                    vegadresse = lagVegadresse(98764L),
                ),
```